### DISCUSSION TITLE
The enlist and tab character

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#5949] [Q&A]
In https://tiddlywiki.com/prerelease/ create a new tiddler as follows

```
<$vars xx="a,	b,	c,	d">
<$list filter="[enlist<xx>]">

</$list>
</$vars>
```
Note that entries are **separated by comma followed by a tab**. Enlist produces:

```
a,
b,
c,
d
```

Is this intentional? Does enlist use tab as delimiter to produce a list of title or this is a bug?

**Comment by kookma** [comment]
I forgot, tab is a filter whitespace! https://tiddlywiki.com/prerelease/#Filter%20Whitespace

So if I want to keep tabs, I have to enclose entries in `[[...]]`

**Comment by pmario** [comment]
See: http://www.regular-expressions.info/shorthand.html  the 3rd paragraph. ... tab is a whitespace
